ctapipe/tools/extract_charge_resolution.py

```diff
@@ -19,36 +20,35 @@
 
 class ChargeResolutionGenerator(Tool):
     name = "ChargeResolutionGenerator"
-    description = "Generate the a pickle file of ChargeResolutionFile for " \
-                  "a MC file."
```

Comment:
Again, this is not specific to this PR, I just realized it again, while reading this diff. 

As we all know, it is quite fundamental in Python to describe a class or function using a *docstring*. Docstrings are so fundamental in Python that they are introduced in the [Python Tutorial at the same time when they explain how to define a function](https://docs.python.org/3/tutorial/controlflow.html#defining-functions). 

So in a readers mind the line above might spark questions like:

 * Do all Tools need a member named `description`
 * Why is no docstring used for this?
 * Is it safe to add a docstring to a `Tool` or will it somehow be modified using `description`

Again, I think a reader might trip over this and spend some minutes perusing the ctapipe docu and the class definition of `Tool` to understand why `description` is used and not `__doc__`.